examples/slack-connector/src/main/resources/application.properties

```diff
@@ -0,0 +1,47 @@
+########################################################################
+#Properties used by channel connector connector framework.
+########################################################################
+
+#The base URL for Chatbot Runtime.
+sfdc.einstein.bots.runtime-url=${BOT_RUNTIME_URL:https://runtime-api-na-west.stg.chatbots.sfdc.sh}
+
+#The name of the integration that will be used. This should match the Integration Name used
+#when adding connected app API connection to Bot.
+sfdc.einstein.bots.integration-name=${BOT_INTEGRATION_NAME:sfdc.chatbot.api}
+
+#OAuth Properties
+#Path to the private key needed to get OAuth Tokens.
+sfdc.einstein.bots.oauth.private-key-file=${OAUTH_PRIVATE_KEY_FILE_PATH:/app/target/classes/private_key.der}
+
+#The URL to hit to obtain an OAuth Token
+sfdc.einstein.bots.oauth.login-endpoint=${LOGIN_END_POINT:https://login.test1.pc-rnd.salesforce.com/}
+
+#ID of the Connected app you setup for API Connection.
+sfdc.einstein.bots.oauth.connected-app-id=${CONNECTED_APP_ID:3MVG9l3R9F9nGODSUSsBSTRIINiGzUl5y6Fi.G6JMNGh54OFHBADNVvyx5XegzsVkKUss0zXxR78rb0FSYIxT}
+
+#Secret of the Connected app you setup for API Connection.
+sfdc.einstein.bots.oauth.connected-app-secret=${CONNECTED_APP_SECRET:78BF17D1787B59B6D0F598EB0BA406587ADCB379B45A36DAD59E55A9D3668F62}
+
+#The user ID for the login that will be used to obtain oauth tokens (should be the user that created the connected app in Salesforce)
+sfdc.einstein.bots.oauth.user-id=${SFDC_USER_ID:admin1@esw5.sdb3}
+
+#Cache properties
+#Time to cache information to avoid network requests. For e.g oAuth token will cached for ttlseconds.
+sfdc.einstein.bots.cache.ttlseconds=${CACHE_TTL_SECS:600}
+
+#Force endpoint (should be your salesforce instance endpoint)
+sfdc.einstein.bots.force-config-endpoint=${FORCE_CONFIG_ENDPOINT:https://esw5.test1.my.pc-rnd.salesforce.com}
+
+#ORG ID
+sfdc.einstein.bots.orgId=${ORG_ID:00DSB0000001ThY2AU}
+
+#BOT ID
+sfdc.einstein.bots.botId=${BOT_ID:0XxSB00000006qD0AQ}
```

Comment:
THis whole file should be worked on as it includes too many sensitive details. Please refer twitter connector.

Reply:
Removed all of them - made it similar to Twitter connector